docs/src/pages/components/grid/grid.md

```diff
@@ -136,3 +136,7 @@ If used, these props may have undesirable effects on the height of the `Grid` it
 Material-UI doesn't provide any CSS Grid functionality itself, but as seen below you can easily use CSS Grid to layout your pages.
 
 {{"demo": "pages/components/grid/CSSGrid.js", "bg": true}}
+
+## System props
+
+As a CSS utility component, the `Grid` supports all [`system`](/system/properties/) properties. You can use them as prop directly on the component.
```

Comment:
I was wondering about this for https://github.com/mui-org/material-ui/pull/24496. Should it be documented in the API pages?

Reply:
I wasn't sure where to add it to be honest... We could whitelist the components that implement the system props (it should be just four of them) and we can automatically add it on the API page. To be honest, I would maybe add it on both pages if that makes sense to you.

Reply:
Both pages sounds fine

Reply:
Alright, let me create a PR for the buildAPI changes 👍

Reply:
Here it is https://github.com/mui-org/material-ui/pull/24503